indico/MaKaC/webinterface/pages/conferences.py

```diff
@@ -661,22 +662,22 @@ def _extractInfoForButton(self, item):
                 info["minutesLink"] = True
                 info["materialLink"] = True
                 info["cloneLink"] = urlHandlers.UHConfClone.getURL(item)
-            if item.getAccessController().canUserSubmit(self._rh._aw.getUser()):
+            elif item.as_event.can_manage(session.user, 'submit'):
                 info["minutesLink"] = True
                 info["materialLink"] = True
 
         elif itemType == 'Session':
-            session = item.getSession()
-            info['parentProtection'] = session.getAccessController().isProtected()
-            if session.canModify(self._rh._aw) or session.canCoordinate(self._rh._aw):
+            sess = item.getSession()
```

Comment:
Luckily not that many places - I think during the initial flask migration there were only 2 or 3 conflicts in the whole codebase